README.md

```diff
@@ -17,32 +18,28 @@ $ npm install metaschema
 
 #### create(schemas)
 
-  - `schemas: `[`<Iterable>`] schemas in form [name, schema, source] (the
-        'source' is optional)
-
-*Returns:* `[ `[`<MetaschemaError>`]`, `[`<Metaschema>`]` ]`
+- `schemas:`[`<Iterable>`] schemas in form [name, schema, source] (the
+  'source' is optional)
```

Comment:
I would prefer that this line had this line had the same indentation from parameter comment beginning.